### PR TITLE
Update what's new for v5.7.1903 release.

### PR DESCRIPTION
**Pull request by beefo**

### CODE REVIEW

File: src/new.md [hunk 1]
```diff
@@ -4,6 +4,49 @@ permalink: /resources/new/
...
+
+- [DutyStatusLog]({{site.baseurl}}/software/api/reference/#T:Geotab.Checkmate.ObjectModel.DutyStatusLog): Added `EditRequestedByUser`.
+
+- [DutyStatusLog]({{site.baseurl}}/software/api/reference/#T:Geotab.Checkmate.ObjectModel.DutyStatusLog): Locations will not be included with DutyStatusLog by default. To include locations must use `dutyStatusLogSearch.IncludeLocations: true`.
+
+- [DutyStatusLogType]({{site.baseurl}}/software/api/reference/#T:Geotab.Checkmate.ObjectModel.DutyStatusLogType): Added `HosEnabled`, `HosDisabled`.
+
+- [DVIRLog]({{site.baseurl}}/software/api/reference/#T:Geotab.Checkmate.ObjectModel.DVIRLog): Added `RepairDate`.
+
+- [IoxAddOn]({{site.baseurl}}/software/api/reference/#T:Geotab.Checkmate.ObjectModel.IoxAddOn): Added `DateTime` representing when the channel was set to the given value..
```
Comment (thetranman) [inline src/new.md]: Remove extra period.

File: src/new.md [hunk 2]
```diff
@@ -4,6 +4,49 @@ permalink: /resources/new/
...
+
+- Serialization: ISO date time at zero hour will now have full ISO time ex `1986-01-01` -> `1986-01-01T00:00:00.000Z`.
+
```
Comment (thetranman) [inline src/new.md]: Missing:

SecurityRole

"Added HardwarePurchase."

Comment (beefo) [reply src/new.md]: Intentionally left out. Mistakenly thought SecurityRole was supported but is not.

File: src/new.md (hunk 1 above)
Comment (beefo) [reply src/new.md]: Thanks. Fixed.

**Pull request merged**